application/utils.py

```
@@ -18,21 +18,45 @@
+# chat-gpt identified headers that are safe to log
+allowed_headers = frozenset({
+    'Host',
+    'User-Agent',
+    'X-Forwarded-For',
+    'X-Real-IP',
+    'Content-Type',
+    'Content-Length',
+    'Accept',
+    'Accept-Encoding',
+    'Accept-Language',
+    'Connection'
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)